### PR TITLE
Improve dtab playground error message

### PR DESCRIPTION
**Pull request by adleong**
If the Path or Dtab is invalid, display a better error message than "Bad Request"

### CODE REVIEW

**Comment by gtcampbell** [comment]
:star: code looks good; what would it take to add a test for this?

**Comment by rmars** [comment]
lgtm!

File: admin/src/main/scala/io/buoyant/admin/names/DelegateApiHandler.scala [hunk 1]
```diff
@@ -177,9 +166,11 @@ object DelegateApiHandler {
     def writeBuf[T](t: T): Buf = Buf.ByteArray.Owned(mapper.writeValueAsBytes(t))
   }
 
-  def getDelegateRsp(dtab: String, path: String, delegator: Delegator): Future[Response] =
-    (dtab, path) match {
-      case (DtabStr(d), PathStr(p)) =>
+  def getDelegateRsp(dtab: String, path: String, delegator: Delegator): Future[Response] = {
+    val dtabTry = if (dtab == null) Return(Dtab.empty) else Try(Dtab.read(dtab))
```
Comment (olix0r) [inline admin/src/main/scala/io/buoyant/admin/names/DelegateApiHandler.scala]: might be a little robust with something like:
```scala
dtab match {
  case null | "" => Return(Dtab.empty)
  case dtab => Try(Dtab.read(dtab))
}
```

**Comment by olix0r** [comment]
lgtm

**Pull request merged**